### PR TITLE
Add Buck sample.

### LINKED ISSUE #57: Add sample buck project to the repo

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #57.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,14 @@ before_install:
   - sudo apt-get install clang --quiet --yes
   ## GCC.
   - sudo apt-get install build-essential --quiet --yes
+  ## Buck.
+  - sudo apt-get install ant python git
+  - git clone https://github.com/facebook/buck.git
```
Comment (artem-zinnatullin) [inline .travis.yml:30]: Yeah, there is no built version of buck for Linux/Windows so you have to build it yourself…

Comment (arturdryomov) [reply .travis.yml:30]: You are kidding, right? Maybe at least make it a single thing using `&&`?

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (artem-zinnatullin) [reply .travis.yml:30]: It'll be harder to read, we have comments as dividers.

**Review by arturdryomov — APPROVED** [review]

**Comment by artem-zinnatullin** [comment]
Nah, later

**Pull request closed without merging**